crates/frontend/src/api.rs

```diff
@@ -56,11 +83,12 @@ pub async fn fetch_snapshots(
     tenant: String,
     page: i64,
     count: i64,
+    all: bool,
```

Comment:
@mahatoankitkumar , can we either pass `ListFilters` struct or its properties individually as parameters to all these api functions. I would prefer individual parameters but I am okay with anything as long as it is consistent.

Why I prefer individual parameters?
As we won't have to construct a dummy ListFilters struct instance to pass the values which is being done at some places, I can directly write `0` or `true`, etc, and also removes the baggage of cloning if we want to reuse the values, as these primitive types implement Copy and its also cheap (not really a deal breaker).

Removing the baggage of cloning a dummy struct instance just because we want to reuse the values is a deal breaker for me.
https://github.com/juspay/superposition/pull/209/files#diff-e4e1d9e001dbf707cd00c7d34f98e60a017d96fd2be2230a10e5e625307819b4R46-R54

Reply:
For this we can raise a different PR to make it consistent.
I'll raise the next PR on this, no worries.